Sources/DepoCore/Helpers/Optional+Convenience.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright © 2021 Rosberry. All rights reserved.
+//
+
+public extension Optional {
+    var array: [Wrapped] {
+        map { wrapped in
+            [wrapped]
+        } ?? []
+    }
+
+    func map<T>(keyPath: KeyPath<Wrapped, T>) -> T? {
```

Comment:
Which case of using this func? 
`optional?.value` will gives same effect isn't it?